src/core/utils/helpers/general.ts

```diff
@@ -345,4 +345,19 @@ export const dataIsNotEmpty = (data: unknown[]) => Boolean(data.length);
 export const constructModalId = (prefix: string, fragments: string[]) =>
   `${prefix ? `${prefix}-` : ""}${fragments.join("-")}`;
 
+export const hasNumberInSeries = (series: Work["series"]) => {
+  // Filter out series that don't have a 'numberInSeries' property
+  const seriesWithNumber = series.filter(
+    (serie) => serie.numberInSeries?.number
+  );
+
+  // Sort the series based on their 'numberInSeries' property
+  return seriesWithNumber.sort((a, b) => {
+    if (a?.numberInSeries?.number && b?.numberInSeries?.number) {
+      return Number(a.numberInSeries.number) - Number(b.numberInSeries.number);
+    }
+    return 0;
+  });
+};
```

Comment:
Please reconsider the this function:

1. The naming is off. Functions starting with `has` or `is` typically return a boolean. That is not the case here. Better examples would be `numberedSeries` or `getNumberedSeries`.
2. Personally I do not understand *why* we need to do any sorting. It is not mentioned anywhere in the requirements. As I see it we should be fine working with any numbered series entry.
3. This function is placed in `utils/helpers/general.ts`. It could not get any more generic :smile:. It would be good to move all our work/manifestation helpers to a separate file. Note that moving this around may be outside the scope of this PR.